src/_includes/head.html

```diff
@@ -66,6 +66,15 @@
     <link href="{{css}}" rel="stylesheet" type="text/css">
   {% endfor -%}
 
+  <!-- Material Icon Styles -->
+  <style>
+    .material-icons.md-18 { font-size: 18px; }
+    .material-icons.md-24 { font-size: 24px; }
+    .material-icons.md-36 { font-size: 36px; }
+    .material-icons.md-48 { font-size: 48px; }
+  </style>
+
+
```

Comment:
So actually we don't want to include styles here in the header and this won't be consistent with our current external link styles.

```suggestion
```

Reply:
Ok I will make the changes

Reply:
@parlough I have made the changes that you have requested. 
```
span {
  vertical-align: middle;
  margin-left: 2px;
  font-size: 12px;
  color: inherit;
  opacity: 0.8;
}
```

Instead of using styles in the above way, I have applied the styles to the` .material-icons` class as applying styles in span would result in rendering styles for every span tag that uses the `_sidebar.css` file.